packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.spec.ts

```diff
@@ -17,10 +17,10 @@ import { getSpecTypeByNamedList } from '../../utilities/tests/parameterized';
 
 async function setup(): Promise<Fixture<AnchorTabs>> {
     return fixture<AnchorTabs>(
-        html`<nimble-anchor-tabs activeid="tab-2">
+        html`<nimble-anchor-tabs activeid="tab-two">
```

Comment:
Great investigation! I'm a bit surprised that fast is mutating the id attribute. I can see they are doing it to [make sure they setup aria association correctly](https://github.com/microsoft/fast/blob/f8dde59eee21a1152263447d22a76593ee5ed9e5/packages/web-components/fast-foundation/src/tabs/tabs.ts#L195).

But still, id is a user facing attribute and must be globally unique across the whole page, not just within FAST's usages within a component. So that's a much higher bar and I think you've just shown that their uniqueness algorithm is not sufficient and is prone to collisions / can easily suffer collisions.

We need some kind of guards for our clients, I can easily see this issue happening to them. We can create maybe a new issue or tech debt issue to think about how to address it. I don't want to block this PR in-specific on it so we can get the intermittency fixed.

Some ideas for the separate issue / tech debt:
- We file an issue upstream to FAST to explain how easy a collision was and to get them to make uniqueid better
- We change our uniqueid usage in anchor tabs, but that doesn't fix potential issues in tabs
- We may need to audit to see what other controls may face the same issues
- Alternatively we put restrictions on our users, i.e. make a policy where we encourage users to not set ids or report warnings if ids are set to collidable values or something (not sure how practical that idea is)

Reply:
Created issue #1043